.github/workflows/ci.yml

```diff
@@ -15,24 +15,25 @@ jobs:
     strategy:
       matrix:
         include:
-          # Dart framework may contain breaking changes in minor version releases, not following semver.
-          - name: Dart 3.0, Ubuntu
+          # Dart 3.1
```

Comment:
I think the main reason is the resource limitation from github for actions.

As you know, running on non-Linux operating systems requires more [resources](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits).

In general, there is no need to test all versions on all operating systems because the reason for this testing is not to evaluate the internal package structure. The reason for this action is to examine package execution, development constraints, platform limitations, and local variables.

Reply:
Could it be that these "development constraints, platform limitations, and local variables" change depending on the combination of Dart version and OS platform? In other words, is it possible that the Parse SDK works with Dart 3.0 on Windows but not with Dart 3.1 on Windows?

Reply:
No, nothing like that happens and there is no problem.

Reply:
I'll try this out real quick in https://github.com/parse-community/Parse-SDK-Flutter/actions/runs/6398999438 and revert if all tests pass.